cypress/integration/regressions.spec.js

```diff
@@ -40,11 +40,6 @@ context('Regressions', () => {
 
   // https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/158
   it('should display relevant English results', () => {
-    cy.visitSearch('see')
-    cy.get('[data-cy=search-results]')
-      .should('contain', 'wâpiw')
-      .and('contain', 'wâpahtam')
-      .and('contain', 'wâpamêw')
```

Comment:
@aarppe @eddieantonio wapiw has translation "s/he sees, s/he has vision CW", with the current runtime keyword extraction, you'll have to type in "sees" have the result.

The English stemming/lemmatization can be improved by an English FST

Reply:
@aarppe sorry, after some thinking I realized this is really bad. I'll add a dumb English stemmer real quick, in the process of English keyword extraction

Reply:
@aarppe @eddieantonio now a stemmer is used and things goes well :)